### PR TITLE
Refactor: Edit file structure

### PR DESCRIPTION
**Pull request by bharati-21**
Create separate folders for components and pages.
- Imported styles inside the component 
- Removed style imports from the main stylesheet

Please note that I have edited the imports in the common file to simply be imports of the form `export {xyz} from abc` in another branch and that is yet to be reflected.

### CODE REVIEW

File: src/components/Header/header.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .parallax-image {
-    background: url('../assets/images/library-img.jpg') no-repeat center center/ cover; 
+    background: url('../../assets/images/library-img.jpg') no-repeat center center/ cover;
```
Comment (ravikumar0403) [inline src/components/Header/header.css:2]: You have used path alias in this project ig, will that work here? not sure, try it out and let me know :P

**Review by ravikumar0403 — APPROVED** [review]
Looks good. Just a suggestion, if you are using css imports in each component separately then you can try out css modules.

**Review by anuj-kumary — APPROVED** [review]
- Good folder structure and clean code
- Nice naming conventions

File: src/components/Header/header.css (hunk 1 above)
Comment (bharati-21) [reply src/components/Header/header.css:2]: Sure. Since it was in another branch, I did not use it here.

**Comment by bharati-21** [comment]
> Looks good. Just a suggestion, if you are using css imports in each component separately then you can try out css modules.

Yes, I shall work on it once I'm done with P0 features. 
Thank you for your review!

**Comment by bharati-21** [comment]
> * Good folder structure and clean code
> * Nice naming conventions

Thank you for your review!

**Pull request merged**